api/routers/areas.py

```diff
@@ -36,6 +36,13 @@ async def get_area(area_id: str):
     """
     Returns the configuration related to the area <area_id>
     """
+    if area_id.upper() == "ALL":
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
```

Comment:
Why the `get` operation is forbidden in the area "ALL"?
I thought that we will allow all the read operations over areas :thinking:

Reply:
This operation should be valid, and return an area with all cameras

Reply:
But this will not validate the `response_model` set in:
`@areas_router.get("/{area_id}", response_model=AreaConfigDTO)`
Pydantic expects to return only one area, and if we give the ID: "All" a ValidationError from Pydantic is raised.

That was the reason why I think a good alternative was to inform the user that there is an endpoint to list all the cameras.
"Area with ID: 'ALL' is not valid. Instead, try the endpoint GET /areas to get all areas."

Reply:
Hmm, but does ALL refer to "one area with all cameras" or "all areas"? I think it is the first. Which is not necessary the same as the second

Reply:
ALL refers to "one area with all cameras". So if you do GET /areas/ALL you should get the hardcodded information of an inexistant area "id: ALL, name: ALL, etc" that inside cameras contains all camera ids

Reply:
Okey, I understand ALL means all the areas. I change it. Now, ALL means all the cameras in an area called "All".